git_fat/git_fat.py

```diff
@@ -584,13 +603,14 @@ def read_credentials_from_file(self, file_path, section):
         parser = cfgparser.SafeConfigParser()
         parser.read(file_path)
         try:
-            return parser.get(section, AWS_S3Backend.AWS_ACCESS_KEY), parser.get(section, AWS_S3Backend.AWS_ACCESS_SECRET_KEY)
+            return (parser.get(section, AWS_S3Backend.AWS_ACCESS_KEY),
+                    parser.get(section, AWS_S3Backend.AWS_ACCESS_SECRET_KEY))
         except cfgparser.NoSectionError:
             raise RuntimeError(
                 "AWS authentication file '%s' does not have section '%s'" % (file_path, section))
         except cfgparser.NoOptionError:
-            raise RuntimeError("AWS authentication file section '%s' does not have proper keys setup (%s, and %s)" %
-                               (file_path, AWS_S3Backend.AWS_ACCESS_KEY, AWS_S3Backend.AWS_ACCESS_SECRET_KEY), section)
+            raise RuntimeError("AWS authentication file section '%s' does not have proper keys     setup (%s, and %s)" % (file_path, AWS_S3Backend.AWS_ACCESS_KEY,
```

Comment:
This won't log out the actual AWS_ACCESS_KEY will it?

Reply:
I'm not entirely sure how the login/logout actions of boto work, to be honest. You pass credentials into boto3 or you it can use the environment variables set for your workstation. I don't think that using boto would log you out of a different profile.